
Bombay hospitality Ltd. operates a franchise model for producing exotic Norwegian dinners throughout New England. The operating cost for a franchise in a week (W) is given by the equation W = $1,000 + $5X, where X represents the number of units produced in a week. Recent feedback from restaurant owners suggests that this cost model may no longer be accurate, as their observed weekly operating costs are higher.

Objective:
To investigate the restaurant owners' claim about the increase in weekly operating costs using hypothesis testing.

Data Provided:
•	The theoretical weekly operating cost model: W = $1,000 + $5X

•	Sample of 25 restaurants with a mean weekly cost of Rs. 3,050

•	Number of units produced in a week (X) follows a normal distribution with a mean (μ) of 600 units and a standard deviation (σ) of 25 units


Assignment Tasks:
1. State the Hypotheses statement:
2. Calculate the Test Statistic:



state the hypothesis Statement

Definition:
A hypothesis is a statement or assumption that is tested through statistical analysis to determine whether it is true or false.


for ex:

Does drinking coffee improve cognitive function?

- **Null Hypothesis (H0 **): Drinking coffee has no effect on cognitive function.
-** Alternative Hypothesis (H1)**: Drinking coffee improves cognitive function.



2.Calcluate the test Statistic:

**Steps**

1.import requried libraries

2. given values should arrange

3. 3.1:
 caluclate theoretical cost based on model

 formula:
   
   w=1000+5*x

  3.2:

  Calculate standard deviation of cost

  3.3:
  
  Standard Error

  3.4:

 Calcluate-Statistic

 3.5:

 Determine critical value for  one-tailed test

 3.6:

 Decision


In [2]:

import math
from scipy.stats import norm
sample_mean=4050
x_mean=600
x_std_size=25
sample_size=25
aplha=0.05
theoritical_mean=1000+5*x_mean
cost_std_dev=5*x_std_size
print(theoritical_mean)
print(cost_std_dev)

4000
125


In [4]:
SE=cost_std_dev/math.sqrt(sample_size)
print(SE)
z_statistic=(sample_mean-theoritical_mean)/SE
print(z_statistic)
z_critical=norm.ppf(1-aplha)
print(z_critical)

25.0
2.0
1.6448536269514722


In [5]:
if z_statistic>z_critical:
    print("Reject the null hypothesis")
else:
    print("Fail to reject the null hypothesis")

Reject the null hypothesis


**Conclusion :**

The theoretical weekly cost according to the model is Rs. 4000 (W = 1000 + 5 × 600).

The observed sample mean cost from 25 restaurants is Rs. 4050.

The Z-statistic calculated is 2.0, which is greater than the critical value of 1.645 at a 5% significance level (one-tailed test).

There is sufficient statistical evidence at the 5% significance level to support the restaurant owners' claim that the actual weekly operating costs are higher than those predicted by the original cost model. This suggests that the company should consider revising the cost model to better reflect the current operating conditions